doc/languages-frameworks/vim.section.md

```diff
@@ -263,7 +263,7 @@ Sometimes plugins require an override that must be changed when the plugin is up
 
 To add a new plugin:
 
-  1. run `./update.py` and create a commit named "vimPlugins: Update",
+  1. run `./update.py --commit`,
```

Comment:
Similar to the discussion in #83119, I think we should just do this by default. We could keep a `--no-commit` flag, but I don't think its necessary.